pom.xml

```diff
@@ -75,6 +75,9 @@
 		<version.archunit>0.13.0</version.archunit>
 		<version.mockito>3.2.4</version.mockito>
 		<version.junit.mockito>3.2.4</version.junit.mockito>
+    <!-- byte buddy 1.9.7+ is needed to solve dependency errors with spring mock tests see LdapClientTest -->
+    <version.byte-buddy>1.9.7</version.byte-buddy>
```

Comment:
Just to understand: Since we use a different mockito version than provided by spring-boot we have to do this, right?

Reply:
more research is needed. it just works that way.